plugin/trino-prometheus/src/test/java/io/trino/plugin/prometheus/TestPrometheusIntegrationStatus.java

```diff
@@ -106,4 +106,14 @@ public void testPushDown()
         MaterializedResult results = computeActual("SELECT * FROM prometheus.default.up WHERE timestamp > (NOW() - INTERVAL '15' SECOND)");
         assertThat(results).hasSizeBetween(1, 2);
     }
+
+    @Test
+    public void testPrometheusWithoutLabels()
```

Comment:
Please rename to `testSelectWithoutLabels`. 

By the way, when I run only this test in this class, it always fail by "Table 'prometheus.default.up' does not exist".

Reply:
Thanks,I will fix these problems. And can you tell me what is the right way to execute trino unit tests? I execute "mvn test" but aways failed.

Reply:
You can just execute on IDE. We recommend using IntelliJ.

Reply:
Sorry, `TestPrometheusIntegrationStatus` seems not suitable. Cold you move to `TestPrometheusIntegrationMetrics` and use `assertQuery` method?

Reply:
Sure, i will move it

Reply:
`Caused by: org.jdbi.v3.core.statement.UnableToCreateStatementException: org.h2.jdbc.JdbcSQLSyntaxErrorException: Schema "PROMETHEUS" not found; SQL statement:
SELECT timestamp, value FROM prometheus.default.up [90079-200] [statement:"SELECT timestamp, value FROM prometheus.default.up", arguments:{positional:{}, named:{}, finder:[]}]
	at org.jdbi.v3.core.statement.SqlStatement.internalExecute(SqlStatement.java:1770)
	at org.jdbi.v3.core.result.ResultProducers.lambda$getResultSet$2(ResultProducers.java:64)
	at org.jdbi.v3.core.result.ResultIterable.lambda$of$0(ResultIterable.java:54)
	at org.jdbi.v3.core.result.ResultIterable.stream(ResultIterable.java:228)
	at org.jdbi.v3.core.result.ResultIterable.collect(ResultIterable.java:284)
	at org.jdbi.v3.core.result.ResultIterable.list(ResultIterable.java:273)
	at io.trino.testing.H2QueryRunner.execute(H2QueryRunner.java:197)
	at io.trino.testing.QueryAssertions.assertQuery(QueryAssertions.java:161)`
When i use assertQuery("SELECT timestamp, value FROM prometheus.default.up") it will throw 'dbcSQLSyntaxErrorException: Schema "PROMETHEUS" not found'. It seems that H2QueryRunner cause it.
Can i use assertQuerySucceeds() to instead?

Reply:
You need to specify the 2nd argument likes `assertQuery(..., "VALUES ...")`

Reply:
It's worked, thanks for your advice and patience.